### ISSUE TITLE
Site Section Manager: Ability to incorporate pending review process for some users

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/Roles-Management#6]
Hello,
So I've trained a few people so far and ran into a few small issues with the site section manager role. Currently all developers have publishing rights to update to the live site, and I would rather have the ability to incorporate a "pending review" process for the majority of our developers where they would make any changes and then let me know they have made that change and I would go in and publish the page. 

Could we have the ability when I'm creating a user to have them be a site section manager either with publishing rights (there are a few developers who will need publishing rights) or the ability to just be able to edit the pages without publishing rights?

I thought I could get around this by having them save the page they are working with as a draft and not clicking on the update button, and then sending us an email asking us to review, but when they save the page as a draft that page becomes inoperable on the website and that draft page becomes a 404 page until the page has been published.

**Comment by joelworsham** [comment]
There currently is no mechanism in WordPress to have changes to a live page become pending until approved. Pages can only have their status changed to publish, pending, draft, or trashed.

This kind of mechanism would need to be added. I'll have to look into this.

**Comment by brashrebel** [comment]
It might be possible to incorporate the staging site somehow. If someone wants to propose changes to content they can safely do it there. Otherwise there isn't an existing way to make a page published and pending at the same time. I recommend using staging for now and that we explore a more elegant solution in the future.

**Comment by banksashleym** [comment]
Ahh, gotcha...The way we are going about it now will work until I get back. I've been very selective in the people that I've chosen to be developers and I feel comfortable with them making changes to the live site. But when I do get back I would like to explore some additional options - we don't really want to give free reign for everyone to be able to update right to the live site without someone in marketing approving the content.

**Comment by brashrebel** [comment]
I completely agree with you. We'll want to find that balance of empowering people to do more on their own without your help but not granting them permission and access to do things they shouldn't. There's a lot of ways to improve that so I look forward to those discussions.

**Comment by banksashleym** [comment]
I think that incorporating the staging side will actually solve our problem. We can schedule a time to discuss this when I come back if that's possible? That way for those people I don't feel comfortable making changes to the live site i can just make them users on the staging, but for those that do need access to be able to publish I can add them as users directly to the live side. 

Would the process be easy on how to selectively publish pages/content from the staging to the live? Or is that something I would have to copy/paste from the staging to the live?

**Comment by brashrebel** [comment]
Currently there is no mechanism available for selectively pushing some content from staging to production. If it were necessary we could explore it. Otherwise, yes you'll have to move it manually.
